### DISCUSSION TITLE
Only one fan speed showing for a dual fan AIO

### DISCUSSION THREAD

**Discussion by helixbeenDRANKIN** [Rem0o/FanControl.Releases#1150] [General]
Recently downloaded the porgram and love its functionality but I unfortunately only see one fan speed showing for my AIO but its a 240mm with two fans as well as one case fan showing. Not sure what the issue is but any assistance would be greatly appreciated!

![fan control gyazo](https://user-images.githubusercontent.com/108300638/193986537-4771c707-53d0-43eb-a143-d06665ae8c28.png)

**Comment by Rem0o** [comment]
FanControl doesn't show "fans", it shows headers.  Your 2 fans are most likely plugged onto the same channel.
